libcodechecker/libhandlers/server.py

```diff
@@ -247,6 +252,28 @@ def add_arguments_to_parser(parser):
                                 help="Stops all of your running CodeChecker "
                                      "server instances.")
 
+    database_mgmnt = parser.add_argument_group("Database management arguments.",
+            """ WARNING these commands needs to be called with the same workspace
+            and configuration arguments as the server was started.""")
+    database_mgmnt = database_mgmnt. \
+        add_mutually_exclusive_group(required=False)
+
+    database_mgmnt.add_argument('--status',
+                                type=str,
+                                dest="status",
+                                action='store',
+                                required=False,
+                                help="Name of the product to get "
+                                     "the database status for.")
+
+    database_mgmnt.add_argument('--upgrade-schema',
```

Comment:
Why do we want to have a commend for this? Shouldn't this be automatic?

Reply:
The schema upgrade will be automatic only for the configuration database, for the run database it will be always a manual.